server/routerlicious/packages/services-shared/src/restLessServer.ts

```diff
@@ -99,8 +108,9 @@ export class RestLessServer {
     }
 
     private async parseStreamRequestFormBody(request: IncomingMessageEx): Promise<void> {
+        const formidable = (await inclusion("formidable")).default;
```

Comment:
Does this mean that the `formidable` version we are bumping to does not allow what we used to have, like `import formidable from "formidable";`?

If that is the case, I'm curious: why do we need to use `inclusion` instead of doing something like
```ts
// eslint-disable-next-line @typescript-eslint/no-require-imports
import formidable = require("formidable");
```

Reply:
This is due to Formidable switching to ESModules in v3.0.0. It's a really annoying problem, where packages that use ESModules cannot be imported using `require()`. You would think that `import formidable as "formidable"` would work, but Typescript is transpiling our code into CommonJS which uses `require` wherever there is an `import` (take a look at the `dist/` folders locally). @tylerbutler helpfully pointed us to a [forum thread with possible fixes](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) (none worked) and the [`inclusion` hack](https://github.com/davidmarkclements/inclusion), which does work.

Reply:
Thanks for all the details!